src/main/java/seedu/duke/commands/member/DeleteTeamMemberCommand.java

```diff
@@ -13,9 +16,10 @@ public DeleteTeamMemberCommand(int memberIndex) {
     }
 
     @Override
-    public String executeCommand(ProjectList projects) {
-        String isNamePresent = (TeamMemberList.isMemberPresent(memberIndex)) ? "Member removed" : "Member not present";
-        TeamMemberList.removeTeamMember(memberIndex);
+    public String executeCommand(ArrayList<Project> projects) {
+        boolean isMemberPresent = memberIndex < Project.members.size();
+        String isNamePresent = (isMemberPresent) ? "Member removed" : "Member not present";
```

Comment:
isNamePresent should be renamed. It's to serve as an output.

Reply:
thanks sean!